src/stores/notification.rs

```diff
@@ -29,8 +29,8 @@ pub trait NotificationStore {
         client_id: &str,
         payload: &MessagePayload,
     ) -> stores::Result<Notification>;
-    async fn get_notification(&self, tenant_id: &str, id: &str) -> stores::Result<Notification>;
-    async fn delete_notification(&self, tenant_id: &str, id: &str) -> stores::Result<()>;
```

Comment:
Seems that's not from #196 or any related to its PRs, was changed somewhere else.